_data/simple-icons.json

```diff
@@ -4621,6 +4621,11 @@
             "hex": "782A90",
             "source": "https://github.com/vert-x3/.github/blob/1ad6612d87f35665e50a00fc32eb9c542556385d/workflow-templates/vertx-favicon.svg"
         },
+        {
+            "title": "Ecosia",
+            "hex": "008009",
```

Comment:
```suggestion
            "hex": "5DD25E",
```

It's not the color in their favicon but `#5dd25e` looks like the primary green they are using (now?) on their website:

![image](https://github.com/user-attachments/assets/72fee163-64ed-4257-8cbc-7189230ade31)

Reply:
Where are you finding that dropbox press kit?

Reply:
It's not in their press kit or on their website?

Reply:
> the text logo is not their “icon” 

Sure but we don't just use "icons", we use their logo. If we have a press kit available and it doesn't include this logo, we should use the logo they do provide. It seems like they rebranded from the tree icon two years ago but I guess kept some of it around?

Reply:
Do you have a vector version of it besides an outdated favicon file?

Reply:
I mean that as we can't find an official use of the favicon in addition to the [safari-pinned-tab.svg](https://www.ecosia.org/static/icons/safari-pinned-tab.svg) I suggest that we use the wordmark logo displayed here https://commons.wikimedia.org/wiki/File:Logo_Ecosia_2022.svg that is used in the navbar on the main page and in their brand guidelines as a png.

![image](https://github.com/user-attachments/assets/8f23a9fc-13ce-44fd-8ac4-01d9550fdd86)

Reply:
Sorry to raise this again but I think you were originally right! Let's bring back `#008009` here as that's the color used in the Wikimedia Commons file (unless we have any objections/other reasons?).